### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by Blfrg** [mubaidr/vuejs-extension-pack#15]
> Cool. If you have more suggestions go ahead. 
> I am planning to update and publish new version. Since tooling has improved a lot since last version.
> _Originally posted by @mubaidr in https://github.com/mubaidr/vuejs-extension-pack/issues/14#issuecomment-1172486838_

As requested for suggestions, I decided to open an issue to discuss options and allow others to contribute.

---

The pack already includes:
>[Vue 3 Snippets](https://marketplace.visualstudio.com/items?itemName=hollowtree.vue-snippets) - A Vue.js 3 And Vue.js 2 Code Snippets Extension
>by @hollowtree

But this extension hasn't been updated in 2 years, and the repo still says "Vue 2" even though the Marketplace lists "Vue 3".

There's a similar extension with ~1.5m installs:
> [Vue VSCode Snippets](https://marketplace.visualstudio.com/items?itemName=sdras.vue-vscode-snippets) - Snippets that will supercharge your Vue workflow
> by @sdras

This one is more recently updated and may be more actively maintained.
I'm unsure if the pack should prefer one over the other, but I wanted to highlight a possible alternative.

---

I noticed that the extension which is already in the pack:
> [TypeScript Vue Plugin (Volar)](https://marketplace.visualstudio.com/items?itemName=vue.vscode-typescript-vue-plugin)
> by @johnsoncodehk

says:
> It's recommended to [use take over mode instead of this plugin](https://github.com/johnsoncodehk/volar/discussions/471).

Although that looks like a deprecation warning, it means to disable the native TS language server in VS Code while using the extension, which allows the Volar language server to "take over" handling regular TS files and Vue SFC files with TS.
The instruction is more clearly stated on the [Official Vue Page](https://vuejs.org/guide/typescript/overview.html#takeover-mode)
The configuration change is significant. Otherwise, two language servers are parsing TS, which impacts performance.
Maybe the extension pack should call attention to the instructions since the extension cannot automate the change.

---

Since the update will be forward-looking, maybe the extension pack should include Official Vite and Vitest extensions:
> [Vite](https://marketplace.visualstudio.com/items?itemName=antfu.vite) - VS Code for Vite
> by @antfu

> [Vitest](https://marketplace.visualstudio.com/items?itemName=ZixuanChen.vitest-explorer) - Run and debug Vitest test cases
> by @zxch3n

**Comment by mubaidr** [comment]
Yes, I noticed that too. [Vue 3 Snippets](https://marketplace.visualstudio.com/items?itemName=hollowtree.vue-snippets) is not being maintained, and replaced it with [Vue VSCode Snippets](https://marketplace.visualstudio.com/items?itemName=sdras.vue-vscode-snippets). Update is already published. 

I have also added `vite` vscode extension. `vitest` seems good suggestion, I will add it right now. 

Regarding the `volar` extension configuration, I will add links to instructions in the read me. 

Thanks for the suggestions. Will publish update in few minutes.
